src/content/h3/tags/damage_effect/readme.md

```diff
@@ -1,6 +1,129 @@
+| specific damage | string | This string tells the game to apply this damage type in addition to the general damage, used as a bonus modifier calculated *after* the general damage. For example a 'sniper' damage type could have a 2x bonus against vehicle materials, thus when used as a specific damage with a general damage that has a 1x modifier on vehicles, you would still get the 2x bonus on vehicles from the specific damage
+| custom response label | string | In a [model](~) you can set up responses to play when a part of a unit reaches a certain health threshold due to damage using labels defined here. An example being the ability to set units on fire, the flamethrower or firebomb has the "fire" response label, which then triggers an [instant response](~model) to create a fire effect onto that unit once they've reach a defined % of health due to that damage.
+| AI stun radius | real | When this damage effect is triggered via [projectile](~) detonation, this radius in world units defines how far from the detonation point the 'AI stun bounds' value extends to
+| AI stun bounds | real | Thess values defines how much 'stun" an AI unit is subjected to from this attack, based on range from impact if an AOE attack, if this stun value is greater than or equal to the [character](~) tags 'stun threshold' value, the AI unit enters their 'stunned' behavior and animation state, an example being grunts holding their head and stumbling around
+| shake radius | real | When this damage effect is triggered via [projectile](~) detonation, this radius defines how far from the detonation point a screen shake effect is played on players
+| EMP radius | real | needs additional research
+| AOE spike radius | real | When used for AOE damage effects, this will act as a smaller radius within the entire AOE that triggers the below value damage boost. An example would be an explosive projectile having a max radius of 5 world units and a spike radius of 1 world unit, should a target be hit by the AOE within 1 world unit of the detonation origin, trigger the damage bump
+| AOE spike damage bump | real | When this damage effect effects a target via AOE and the target is within the above fields radius, add this much additional damage
+
+damage response | [damage_response_definition](~) | This field is used to reference the tag that controls screen shake/motion and controller haptics when a target is hit by this damage effect instance.
+
+# As melee
+
+During melee animations, specific frames during the animation are set as 'primary keyframes', which means during these frames in the animation the melee damage effect of the AI is active and will hurt units within their radius, under normal circumstances a melee animation cannot hit a single target more than once, so if you get hit by active frame 1, but are still in the radius of the melee attack for the next 4 active frames, you will still only take one instance of damage.
```

Comment:
"Specific frames of melee animations are set as 'primary keyframes'. During these frames, the melee damage effect is active, and will hurt units within its radius. Under normal circumstances, a melee animation cannot hit a single unit more than once. For example, if you are within the damage radius for 5 primary keyframes, you will only take one instance of damage."